modules/claims_api/app/clients/claims_api/bgs_client/definitions.rb

```diff
@@ -59,6 +59,23 @@ module ClaimantServiceBean
               data: nil
             )
           )
+        module FindAssignedFlashes
+          DEFINITION =
+            Action.new(
+              service: ClaimantService::DEFINITION,
+              name: 'findAssignedFlashes',
+              key: 'return'
+            )
+
+          module AddFlash
+            DEFINITION =
+              Action.new(
+                service: ClaimantService::DEFINITION,
```

Comment:
Where is `ClaimantService::DEFINITION` defined? I am getting `no definition found` errors.

Reply:
@tycol7 This is resolved. Thank you for catching that.